src/app/sb-admin/components/pages/category/category-routing.module.ts

```diff
@@ -0,0 +1,10 @@
+import { NgModule } from '@angular/core';
+import { RouterModule } from '@angular/router';
+import { CategoryComponent } from './category.component';
+@NgModule({
+  imports: [RouterModule.forChild(
+    [{ path: '', component: CategoryComponent }
+  ])],
+  exports: [RouterModule]
+})
+export class CategoryRoutingModule { }
```

Comment:
there should be one parent component above category, framework-manage, framework, publish and term component. and all these component should children of that parent framework mgmt component